lib/DiscoveryStreamFeed.jsm

```diff
@@ -218,6 +217,13 @@ this.DiscoveryStreamFeed = class DiscoveryStreamFeed {
     };
   }
 
+  filterRecommendations(feed) {
+    if (feed && feed.data && feed.data.recommendations && feed.data.recommendations.length) {
+      return {data: this.filterBlocked(feed.data, "recommendations")};
+    }
+    return feed;
```

Comment:
Maybe wrap this in an `else` for clarity?

Reply:
unfortunately doing so fails eslint(no-else-return)

Reply:
Ah...well I guess it's fine in that case.